README.md

```diff
@@ -1,41 +1,67 @@
-# fastly-tools
+# fastly-tools [![CircleCI](https://circleci.com/gh/Financial-Times/fastly-tools.svg?style=svg&circle-token=33bcf2eb98fe2e875cc66de93d7e4a50369c952d)](https://circleci.com/gh/Financial-Times/fastly-tools)
+
+This library is a command line tool for interacting with the FT.com CDN, [Fastly](https://www.fastly.com/).
+
+
+## Requirements
+
+* Node 5.x
 
-Command Line Utility for interacting with fastly
 
 ## Installation
 
-```
-npm install --save-dev @financial-times/fastly-tools
+```sh
+git clone git@github.com:Financial-Times/fastly-tools.git
+cd fastly-tools
+make install
 ```
 
-## Usage
 
-	fastly [options] [command]
+## Development
 
-      Commands:
+### Testing
 
-        deploy [options] [folder]
-           Deploys VCL in [folder] to the specified fastly service.  Requires FASTLY_APIKEY env var
+In order to run the tests locally you'll need to run:
 
-      Options:
+```sh
+make test
+```
 
-        -h, --help  output usage information
+### Install from NPM
 
+```sh
+npm install --save-dev @financial-times/fastly-tools
+```
+
+### Usage
 
-## Commands
+```sh
+Usage: fastly [options] [command]
 
-### deploy
-Deploys VCL in [folder] to the specified fastly service.  Requires FASTLY_APIKEY env var
+Options:
+  -h, --help                 output usage information
+
+Commands:
+  deploy [options] [folder]  Deploys VCL in [folder] to the specified fastly service.  Requires FASTLY_KEY env var
```

Comment:
Can you add a few examples here, please?

From these notes, I think I could run:
- `fastly --help`
- `fastly deploy --env --service {SERVICEID} './vcl'` (using [next-errors](https://github.com/Financial-Times/next-errors) as an example, if I were to run this command from the root dir of that repo)

Reply:
I had originally added an example from next-api but i decided to remove it because i noticed that the package is published to npm so i didnt want to add any private repo links.

Reply:
Is it possible to include an example but use placeholders for the specifics?

Reply:
ok 👍